syslog_ng_exporter.go

```diff
@@ -221,6 +253,10 @@ func main() {
 	log.AddFlags(app)
 	kingpin.MustParse(app.Parse(os.Args[1:]))
 
+	log.Infoln("Starting syslog_ng_exporter", version.Info())
+	log.Infoln("Build context", version.BuildContext())
+	log.Infof("Starting server: %s", *listeningAddress)
+
```

Comment:
Why did you move this up? I think it should at least be below the showVersion check, otherwise it'll print the "starting up" message even when all you've asked for is the version output.